css/css-backgrounds/parsing/border-image-outset-valid.html

```diff
@@ -12,8 +12,10 @@
 </head>
 <body>
 <script>
-test_valid_value("border-image-outset", "1px");
-test_valid_value("border-image-outset", "2");
+// "1px" in Blink, Edge, WebKit; "1px 1px 1px 1px" in Firefox.
+test_valid_value("border-image-outset", "1px", ["1px", "1px 1px 1px 1px"]);
```

Comment:
FWIW, I don't know with which Firefox version have you tested, but Firefox serializes to 1px for me.

Reply:
I observed "1px 1px 1px 1px" using Firefox ESR 52.6.0